### PR TITLE
Bisect auto start

### PR DESCRIPTION
**Pull request by pranitbauva1997**
@chriscool @larsxschneider Please review.

### CODE REVIEW

**Comment by chriscool** [comment]
I am going to Toronto today to give a presentation at the LinuxCon North America there next Tuesday, so I won't have much time these days...

File: builtin/bisect--helper.c [hunk 1]
```diff
@@ -760,6 +763,32 @@ static int bisect_start(struct bisect_terms *terms, int no_checkout,
...
+
+		/*
+		 * TRANSLATORS: Make sure to include [Y] and [n] in your
+		 * translation. THe program will only accept English input
+		 * at this point.
+		 */
+		yesno = git_prompt(_("Do you want me to do it for you "
+				     "[Y/n]? "), PROMPT_ECHO);
+		if (starts_with(yesno, "n") || starts_with(yesno, "N"))
+			exit(0);
```
Comment (chriscool) [inline builtin/bisect--helper.c]: Why `exit(0)` instead of for example `return 0`?

File: builtin/bisect--helper.c [hunk 2]
```diff
@@ -39,6 +39,8 @@ struct bisect_terms {
 	struct strbuf term_bad;
```
Comment (chriscool) [inline builtin/bisect--helper.c]: Missing synopsis update?

Comment (pranitbauva1997) [inline builtin/bisect--helper.c]: Will add.

**Comment by pranitbauva1997** [comment]
Best of luck with your presentation!

File: builtin/bisect--helper.c (hunk 1 above)
Comment (pranitbauva1997) [inline builtin/bisect--helper.c]: I was thinking something at that time. Now I don't remember it. Will try and change it to `return 0` and see if I recall it.

**Pull request closed without merging**